service/nodeconfig/v1/resource/node/create.go

```diff
@@ -59,36 +61,74 @@ func (r *Resource) ApplyCreateChange(ctx context.Context, obj, createChange inte
 		t := types.StrategicMergePatchType
 		p := []byte(UnschedulablePatch)
 
-		{
-			manifest, err := k8sClient.CoreV1().Nodes().Get(n, apismetav1.GetOptions{})
-			if err != nil {
-				return microerror.Mask(err)
-			}
-			fmt.Printf("unschedulable\n")
-			fmt.Printf("%#v\n", manifest.Spec.Unschedulable)
-			fmt.Printf("unschedulable\n")
+		_, err := k8sClient.CoreV1().Nodes().Patch(n, t, p)
+		if err != nil {
+			return microerror.Mask(err)
 		}
+	}
 
-		_, err := k8sClient.CoreV1().Nodes().Patch(n, t, p)
+	var customPods []v1.Pod
+	var systemPods []v1.Pod
+	{
+		fieldSelector := fields.SelectorFromSet(fields.Set{
+			"spec.nodeName": key.NodeName(customObject),
+		})
+		listOptions := apismetav1.ListOptions{
+			FieldSelector: fieldSelector.String(),
+		}
+		podList, err := k8sClient.CoreV1().Pods(v1.NamespaceAll).List(listOptions)
 		if err != nil {
 			return microerror.Mask(err)
 		}
 
-		{
-			manifest, err := k8sClient.CoreV1().Nodes().Get(n, apismetav1.GetOptions{})
-			if err != nil {
-				return microerror.Mask(err)
+		customPods = filterPods(podList.Items, func(p v1.Pod) bool {
+			if p.GetNamespace() == "kube-system" {
+				return false
```

Comment:
Filtering for custom pods.

Reply:
You could do that in single loop w/o filter function and I would find that cleaner.

Reply:
Ok cool. Done.